cg/store/models.py

```diff
@@ -475,7 +475,7 @@ class Sample(Model, PriorityMixin):
     delivered_at = Column(types.DateTime)
     deliveries = orm.relationship(Delivery, backref="sample")
     downsampled_to = Column(types.BigInteger)
-    from_sample = Column(types.String(128))  # DEPRECATED
+    from_sample = Column(types.String(128))
```

Comment:
```suggestion
    from_sample = Column(types.String(128), ForeignKey("sample.internal_id"), nullable=True)
```
Seems like this is a relation-key. N.B! Not tested